docs/manual/user_guide.adoc

```diff
@@ -200,7 +200,11 @@ Looking at the `/tmp/results.xml` file, you will notice lines similar to those b
 === Remediation
 
 ==== Bash Scripts
+
 ComplianceAsCode embeds bash remediation scripts into the SCAP content. This allows for SCAP compatible tools to extract these remediation scripts to aide in potential remediation of system misconfigurations.
+
+A pregenerated Bash remediation script for each profile can be found in `/usr/share/scap-security-guide/bash/` or if you build the project from source in `./build/bash`.
+
```

Comment:
The previous paragraph doesn't mention that the pre-generated scripts are shipped with the package.
And the next paragraph is disconnected from this one, it starts discussing how to generate the scripts yourself.